libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -636,7 +636,13 @@ bool AP_Arming::gps_checks(bool report)
             }
         }
 
-        if (!AP::ahrs().home_is_set()) {
+        // Only check we have an AHRS home when we have at least 1 GPS enabled.
```

Comment:
we'd need a FLIGHT_OPTIONS bit, or a AHRS_OPTIONS bit

Reply:
I've added an AHRS options bit to allow arming without a home.